build/dependencies.props

```diff
@@ -6,7 +6,7 @@
     <InternalAspNetCoreSdkVersion>2.1.0-*</InternalAspNetCoreSdkVersion>
     <MoqVersion>4.7.1</MoqVersion>
     <NETStandardImplicitPackageVersion>$(BundledNETStandardPackageVersion)</NETStandardImplicitPackageVersion>
-    <TestSdkVersion>15.0.0</TestSdkVersion>
-    <XunitVersion>2.2.0</XunitVersion>
+    <TestSdkVersion>15.3.0-*</TestSdkVersion>
```

Comment:
I think it's safe to float. Here's my thinking:

Floating xunit: the next update to NuGet would be `2.3.0-beta3-*` so we won't get automatically lifted off beta2. Also, I used floating this a matter of convenience because the full version of `xunit` and `xunit.runner.visualstudio` differ slightly by their build numbers.

Floating Microsoft.NET.Test.Sdk to `15.3.0-*` should also be ok. We mirror Microsoft.NET.Test.Sdk in our gatekeeper build, so the version on [aspnetcore-ci-dev](https://dotnet.myget.org/feed/aspnetcore-ci-dev/package/nuget/Microsoft.NET.Test.Sdk) is ahead of [the version](https://www.nuget.org/packages/Microsoft.net.test.sdk) on nuget.org. As new nightlies roll in, our gatekeeper build should catch issues before they make it to nuget.org. (hopefully)